services/QuillLMS/engines/evidence/app/models/evidence/automl_model.rb

```diff
@@ -77,51 +75,39 @@ def activate
       false
     end
 
-    def classify_text(text)
-      VertexAI::TextClassifier.run(endpoint_external_id, text)
-    end
+    def classify_text(text) = VertexAI::TextClassifier.run(endpoint_external_id, text)
 
     def older_models
-      @older_models ||= AutomlModel.where(prompt_id: prompt_id).where("created_at < ?", created_at).count
+      @older_models ||=
```

Comment:
I don't believe memoization will work on this multi-line code without `||= begin ... end`, unless something has changed in Ruby.

Reply:
Oh.  Great catch here.